lib/components/map/leg-diagram.js

```diff
@@ -196,7 +197,7 @@ class LegDiagram extends Component {
       mainSvgContent.push(
         <line
           key={`step-${stepIndex}-line`}
-          stroke="#a6a6a6"
+          stroke={grey[400]}
```

Comment:
Extract a constant, declared in this file, for this gray color, and maybe combine with the `gray[300]` because it is also about a line?